src/javascript/static/pages/platforms.js

```diff
@@ -19,8 +21,21 @@ const os_list = [
     // }
 ];
 
+const addLoader = () => {
+    const container = getElementById('content-holder');
+    const content = getElementById('content');
+    const loading_spinner = createElement('div', { html: '' });
+    container.appendChild(loading_spinner);
```

Comment:
why do we need to make this so complicated? let's just add the loading element to the HTML code and then wait for authorize to hide it.

Reply:
sure, I will fix that